account_bank_statement_import_adyen/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2017 Opener BV (<https://opener.amsterdam>)
+# © 2020 Vanmoof BV (<https://www.vanmoof.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Adyen statement import",
+    "version": "12.0.1.0.0",
+    "author": "Opener BV, Vanmoof BV, Odoo Community Association (OCA)",
+    "category": "Banking addons",
+    "website": "https://github.com/oca/bank-statement-import",
+    "depends": [
+        "account_bank_statement_import",
+        "account_bank_statement_clearing_account",
+    ],
+    "data": [
+        "views/account_journal.xml",
+    ],
+    "installable": True,
```

Comment:
Can you add `"license": "AGPL-3",`? Sorry that you have to make up for my laziness of that time!

Reply:
No need to apologize. I also like to have things perfectly done.